js/main.js

```diff
@@ -0,0 +1,187 @@
+    name: 'Yochai Benkler',
+    profession: 'Berkman Professor of Entreprenurial Legal Studies at Harvard Law School',
+    description: 'Benkler studies commons-based peer production, and published his seminal bool The Wealth of Networks in 2006.',
+  },
+  {
+    image: './images/speaker_02.png',
+    name: 'Kilnam Chon',
+    profession: '',
+    description: "Kilnam Chon helped bring the Internet to Asia and is an outspoken advocate for the open web and digital commons. in 2012, he was inducted into the inaugural class of the Internet Society's(ISOC) Internet Hall of Fame",
+  },
+  {
+    image: './images/speaker_03.png',
+    name: 'SohYeong Noh',
+    profession: 'Director of Art Centre Nabi and a board member of CC Korea',
+    description: 'As the main venue for new media art production in Korea, Nabi promotes cross-diciplinary collaboration and understanding among science technology, humanities, and the arts.',
+  },
+  {
+    image: './images/speaker_04.png',
+    name: 'Julia Leda',
+    profession: 'President of Young Pirates of Europe',
+    description: "European ingetration, political democracy and participation of youth through online as her major condern, Reda's report outlining potential chancges EU copyright law was approved by the Parlament in July.",
+  },
+  {
+    image: './images/speaker_05.png',
+    name: 'Lila Tretikov',
+    profession: 'Executive Director of the Wikimedia Foundation',
+    description: 'Lila Tretikov is the Exectuvie Director of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely avaiable un 290 languages and used by nearly half a billion people around the world every month',
+  },
+  {
+    image: './images/speaker_06.png',
+    name: 'Ryan Merkley',
+    profession: 'CEO of Creative Commons, ex COO of the Mozilla Foundation',
+    description: 'Ryan had been leading open-source projects as the Mozilla Foundation such as the open source movement.',
+  },
+];
+
+const printSpeakers = () => {
+  const speakersSection = document.getElementById('featuredSpeakers');
+  speakers.forEach((speaker) => {
+    const ul = document.createElement('ul');
+    ul.classList.add('speakers__list__container');
+    ul.innerHTML = `
+        <li class="featured__speakers__item">
+            <div class="speaker__description__container">
+                <figure>
+                    <img src="${speaker.image}" alt="" class="featured__speaker__image">
+                </figure>
+                <div class="featured__speaker__nameAndProfession">
+                    <h3 class="featured__speaker__name">${speaker.name}</h3>
+                    <h4 class="featured__speaker__profession">${speaker.profession}</h4>
+                    <hr>
+                    <p class="featured__speaker__description">${speaker.description}</p>
+                </div>
+            </div>
+        </li>
+        `;
+    speakersSection.appendChild(ul);
+  });
+};
+printSpeakers();
```

Comment:
As mentioned above the code for printing the programs and the featured speakers follows a similar pattern. Kindly I suggest you create a generic function that takes the data and the target container as parameters. This function could handle the creation of the elements and appending them to the container, reducing code duplication. 😄